reviews/models/mixins.py

```diff
@@ -272,9 +272,10 @@ def reviews_submit_notification(self, context):
         user_subscriptions = get_user_subscriptions(user, event_type)
         context['no_future_emails'] = user_subscriptions['none']
         context['is_creator'] = user == context.get('reviewable').node.creator
+        context['provider_name']= context.get('reviewable').provider.name
```

Comment:
Is `reviewable` guaranteed to be in the context?

If so, then context['reviewable'] is preferred, to make it clear that it's guaranteed.
If not, then this code will need to be changed to handle the case when it's not there.